eslint-config-angular/index.mjs

```diff
@@ -70,22 +74,11 @@ export const configBase = typescriptEslint.config({
     '@typescript-eslint/prefer-function-type': ['error'],
     '@typescript-eslint/unified-signatures': ['error'],
 
-    'array-bracket-spacing': ['error'],
-    'arrow-parens': ['error', 'as-needed'],
-    'arrow-spacing': ['error'],
-    'block-spacing': ['error'],
-    'curly': ['error'],
-    'jsdoc/newline-after-description': ['off'],
-    'key-spacing': ['error'],
+    'prefer-arrow/prefer-arrow-functions': ['off'],
```

Comment:
what is the reason for turning it off here?

Reply:
Removed.